src/common/hooks/use-selected-asset.ts

```diff
@@ -30,9 +30,9 @@ export function useSelectedAsset() {
   const balance = useMemo<string | undefined>(() => {
     if (!selectedAsset) return;
     if (selectedAsset.type === 'stx')
-      return stacksValue({ value: selectedAsset.balance, withTicker: false });
+      return stacksValue({ value: selectedAsset.balance.toString(), withTicker: false });
     if (selectedAsset?.meta?.decimals)
-      return ftDecimals(selectedAsset.balance, selectedAsset.meta?.decimals);
+      return ftDecimals(selectedAsset.balance.toString(), selectedAsset.meta?.decimals);
```

Comment:
i still think this can take a `BigNumber` https://github.com/blockstack/stacks-wallet-web/blob/51e403aa7a93f5b6efeefb6e887f18a572384499/src/common/stacks-utils.ts#L35-L41